woodwork/logical_types.py

```diff
@@ -299,6 +299,9 @@ def transform(self, series):
                     )
         return super().transform(series)
 
+    def __str__(self):
+        return "{}: {}".format(str(self.__class__), self.datetime_format)
```

Comment:
@jeff-hernandez @thehomebrewnerd Do you think maybe we should not have the datetime format be a part of the __str__? It makes sense for the Ordinal one because it is actually a part of the Logical Type. 
- For Datetime however, we have already changed it to Datetime variables (so the format isn't really useful anymore).

Reply:
I agree. The current implementation also doesn't do anything special when there is no format passed in (None), but I personally don't think it's necessary for datetime to print out the format

Reply:
Let's remove it for Datetime.

Reply:
yep not necessary